src/qt/locale/bitcoin_fr.ts

```diff
@@ -2221,35 +2225,35 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Connected peers</source>
-        <translation type="unfinished"></translation>
+        <translation>Pairs connectés</translation>
     </message>
     <message>
         <source>Add new peer</source>
-        <translation type="unfinished"></translation>
+        <translation>Ajouter un nouveau pair</translation>
     </message>
     <message>
         <source>Remove peer</source>
-        <translation type="unfinished"></translation>
+        <translation>Retirer le pair</translation>
     </message>
     <message>
         <source>One try peer</source>
-        <translation type="unfinished"></translation>
+        <translation>Premier pair d&apos;essai</translation>
     </message>
     <message>
         <source>Error</source>
-        <translation type="unfinished">Erreur</translation>
+        <translation>Erreur</translation>
     </message>
     <message>
         <source>Remove Peer</source>
-        <translation type="unfinished"></translation>
+        <translation>Retirer le pair</translation>
     </message>
     <message>
-        <source>Are you sure you want to remove the peer: </source>
-        <translation type="unfinished"></translation>
+        <source>Are you sure you want to remove the peer:</source>
+        <translation>Vous êtes sûr de vouloir supprimer le pair :</translation>
```

Comment:
Any reason not to swap this to "Etes-vous"?

Reply:
Yes, because there is no question mark so it is more appropriate.